R/module_teal_data.R

```diff
@@ -43,8 +45,19 @@ ui_teal_data <- function(id, data_module) {
   checkmate::assert_class(data_module, "teal_data_module")
   ns <- NS(id)
   shiny::tagList(
-    data_module$ui(id = ns("data")),
-    ui_validate_reactive_teal_data(ns("validate"))
+    div(
+      id = ns("wrapper"),
+      data_module$ui(id = ns("data"))
+    ),
+    div(
+      id = ns("input_error"),
+      class = "teal_validated",
+      uiOutput(ns("error"))
+    ),
+    div(
+      id = ns("wrapper_validate"),
```

Comment:
New proposition for `wrapper_validate` name

- `data_validate`
- `input_validate`
- `teal_data_validate`

If we change this name in here also adjust `shinyjs::hide("wrapper_validate")` and `shinyjs::show("wrapper_validate")` in the code